src/components/Table/TransactionTable.js

```diff
@@ -0,0 +1,212 @@
+import React, { PureComponent } from 'react';
+import propTypes from 'prop-types';
+import moment from 'moment';
+import assert from 'bsert';
+import { Table } from '../index';
+import { connectTheme } from '../../utils';
+
+import { TxManager, TxManagerOptions, UXTXOptions } from '@bpanel/bpanel-utils';
+
+import ExpandedDataRow from './ExpandedDataRow';
+
+/*
+ * TransactionTable
+ *
+ * Wrapper around Table to display
+ * transaction related information
+ *
+ * TODO: better logic around setting custom
+ * TxManager options at runtime with props
+ */
+class TransactionTable extends PureComponent {
+  /*
+   * Create a TransactionTable
+   * Initialize a txManager with default options
+   * @constructor
+   *
+   */
+  constructor() {
+    super();
+    this.txManager = TxManager.fromOptions(TxManagerOptions);
+
+    // TODO: use for caching purposes
+    this._headers = null;
+  }
+
+  /*
+   * Get proptypes
+   *
+   * @static
+   * @returns {Object}
+   */
+  static get propTypes() {
+    return {
+      TxManagerOptions: propTypes.object,
```

Comment:
Is the shape of the options consistent? Meaning, will it always accept the same set of props? If so, it would probably be more useful to use `propTypes.shape` rather than object as it's more descriptive and helps when learning about this component.

Reply:
same comment for some of the others below. Transactions might be fine to say that it's an array of objects, but since transactions are quite complex (and self-explanatory since they have their own documentation) maybe not necessary to write that all out, but at least it indicates it's not an array of strings (e.g. could be txids) for example

Reply:
`Is the shape of the options consistent?` - it doesn't need to be consistent as the options are now optional. The docs to the `TxManager` are where one should go to learn about what the shape is, I don't think that the definition should be in more than one place as it could go out of sync more easily that way

Reply:
I added comments around this code block to clarify for people wanting to learn about the interface to this code